File: CS/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace WindowsApplication1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            InitData();
        }
        public void InitData() {
            var dt = new DataTable();
            dt.Columns.Add("ID", typeof(Int32));
            dt.Columns.Add("Trademark", typeof(string));
            for (int i = 0; i < 20; i++)
                dt.Rows.Add(new object[] { i, "Test" + i });
            myGridControl1.DataSource = dt;
        }
    }
}

File: VB/WindowsApplication1/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace WindowsApplication1
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
            InitData()
        End Sub
        Public Sub InitData()
            Dim dt = New DataTable()
            dt.Columns.Add("ID", GetType(Int32))
            dt.Columns.Add("Trademark", GetType(String))
            For i As Integer = 0 To 19
                dt.Rows.Add(New Object() { i, "Test" & i })
            Next i
            myGridControl1.DataSource = dt
        End Sub
    End Class
End Namespace